tardis/conftest.py

```diff
@@ -136,12 +139,7 @@ def config_verysimple():
     return config
 
 @pytest.fixture(scope="session")
-def simulation_verysimple(config_verysimple, atomic_data_fname):
-    atomic_data = AtomData.from_hdf5(atomic_data_fname)
-    if atomic_data.md5 != '21095dd25faa1683f4c90c911a00c3f8':
-        pytest.skip('Need default Kurucz atomic dataset '
-                    '(md5="21095dd25faa1683f4c90c911a00c3f8"')
-
-    sim = Simulation.from_config(config_verysimple, atom_data=atomic_data)
+def simulation_verysimple(config_verysimple, atomic_dataset):
```

Comment:
You are using the `session` scoped atomic dataset fixture here. I believe you want to use `kurucz_atomic_data` which is the one making use of the `deepcopy`, right?